inc/namespace.php

```diff
@@ -16,6 +16,11 @@
  * Kick everything off.
  */
 function bootstrap() {
+	// If the consent api doesn't exist, load the local composer autoload file which should include it.
+	if ( ! class_exists( 'WP_CONSENT_API' ) ) {
+		require_once plugin_dir_path( __DIR__ ) . 'vendor/autoload.php';
```

Comment:
This would probably work for some cases but is still not completely safe as it doesn't guarantee there is a local vendor directory. In addition the package type in its `composer.json` is `wordpress-plugin` - this means Composer will place it at `wp-content/plugins/..` by default. I think the best option is just to return early and trigger an error in this case and maybe show an admin notice.

If it has been added to the plugins directory then it's up to the site admin to activate it.

```suggestion
	// If the consent api doesn't exist, load the local composer autoload file which should include it.
	if ( ! class_exists( 'WP_CONSENT_API' ) ) {
		trigger_error( 'The WP Consent Level API plugin must be installed and activated', E_USER_WARNING );
		return;
```